.github/workflows/ci.yml

```
@@ -1,14 +1,20 @@
 name: Puree
-on: [push]
+on: [pull_request]
```

`push` should be still supported.

```suggestion
on: [push, pull_request]
```